cwms-data-api/src/main/java/cwms/cda/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -881,8 +914,18 @@ private void store(Connection connection, String officeId, String tsId, String u
             }
         }
 
+        if(versionDate != null) {
+            try {
+                CWMS_TS_PACKAGE.call_SET_TSID_VERSIONED(getDslContext(connection, officeId).configuration(),
+                        tsId, "T", officeId);
+            } catch(DataAccessException e) {
```

Comment:
Log the exception at a low level (FINER/FINEST). Also this should trap only the exact SQL error code that is thrown if the TSID doesn't exist

Reply:
done